stage0/src/acpi.rs

```diff
@@ -160,6 +162,12 @@ pub struct DescriptionHeader {
 }
 
 impl DescriptionHeader {
+    #[allow(dead_code)]
+    pub fn signature(&self) -> &str {
+        // Safety: per the ACPI spec, the signature is in ASCII, which is always valid UTF-8.
```

Comment:
nit: I would also mention here that the header will always be validated.

Also, I think it would be nicer to avoid the unsafe code by using from_utf8 and panicking if it failed.

Reply:
Done.